File: diamonds-api/Diamonds.Api/Common/Entities/Board.cs
using System;
using System.Collections.Generic;

namespace Common.Entities
{
    public class Board
    {
        public IEnumerable<Bot> Bots { get; set; }
        public IEnumerable<Position> Diamonds { get; set; }
        public string BoardId { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
    }

}

File: diamonds-bot-example/game/board.py
class Board(object):

    def __init__(self, data):
        self.id = data["id"]
        self.width = data["width"]
        self.height = data["height"]
        self.bots = data["bots"]
        self.diamonds = data["diamonds"]

    def get_bot(self, bot):
        for item in self.bots:
            if item["name"] == bot.name:
                return item

